app/src/main/java/app/accrescent/client/ui/SettingsScreen.kt

```diff
@@ -70,7 +72,11 @@ fun SettingsScreen(modifier: Modifier = Modifier, viewModel: SettingsViewModel =
                 label = stringResource(R.string.require_user_action),
                 description = stringResource(R.string.require_user_action_desc),
                 modifier = Modifier.fillMaxWidth()
-                    .clickable { coroutineScope.launch { viewModel.setRequireUserAction(!requireUserAction) } },
+                    .toggleable(
+                        value = requireUserAction,
+                        role = Role.Switch,
+                        onValueChange = { coroutineScope.launch { viewModel.setRequireUserAction(!requireUserAction) } }
```

Comment:
nit, since I'm adding another review comment anyway: `!requireUserAction` etc. can be replaced with `it` to reduce column width since the toggle value is in scope here.